src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -187,16 +191,36 @@ namespace AppInstaller::CLI::Workflow
                 }
                 else
                 {
-                    entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetPath, {})));
+                    if (isSymlinkCreationSupported)
+                    {
+                        entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetPath, {})));
+                    }
+                    else
+                    {
+                        // Find matching nested installer file and rename the symlink target with the corresponding command alias if available.
+                        for (const auto& nestedInstallerFile : nestedInstallerFiles)
+                        {
+                            if (relativePath == std::filesystem::path{ ConvertToUTF16(nestedInstallerFile.RelativeFilePath) })
+                            {
+                                if (!nestedInstallerFile.PortableCommandAlias.empty())
+                                {
+                                    std::filesystem::path targetWithCommandAlias = targetInstallDirectory / ConvertToUTF16(nestedInstallerFile.PortableCommandAlias);
+                                    Filesystem::AppendExtension(targetWithCommandAlias, ".exe");
+                                    entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetWithCommandAlias, {})));
+                                }
+
+                                break;
+                            }
+                        }
+                    }
                 }
             }
 
-            const std::vector<Manifest::NestedInstallerFile>& nestedInstallerFiles = context.Get<Execution::Data::Installer>()->NestedInstallerFiles;
-
+            // Even if symlinks are not supported, symlink entries are still needed for accurate reporting of the command aliases.
```

Comment:
After discussion, we'll not create these entries with incorrect data, instead we'll handle accordingly by setting a flag in the portable entry.